components/organisms/TopicBox.js

```diff
@@ -24,10 +24,16 @@ export default function TopicBox(props) {
       </div>
       <p>{props.title}</p>
       <br />
+      <p>{props.body}</p>
+      <br />
       <p className="font-bold">Find information about</p>
-      <p>thing 1</p>
-      <p>thing 2</p>
-      <p>thing 3</p>
+      <div className="flex flex-wrap gap-2 pb-1">
+        {props.subtopics.map((d) => (
+          <div className="h-auto w-auto rounded-md border-2 pl-2 pr-2">
+            <p>{d}</p>
+          </div>
+        ))}
+      </div>
```

Comment:
Generally when we're using "map" it's because we are building a list, so let's make this into a `ul` element with `li`s instead of a `div` containing a paragraph.

Reply:
10-4